integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/IngressSteps.java

```diff
@@ -72,13 +72,13 @@ public void sendCloudEventToIngressOfBridgeWithPathAndDefaultHeaders(String test
 
     @When("^send a cloud event to the Ingress of the Bridge \"([^\"]*)\" with path \"([^\"]*)\" is failing with HTTP response code (\\d+):$")
     public void sendCloudEventToIngressOfBridgeWithPathIsFailingWithHTTPResponseCode(String testBridgeName, String path,
-            int responseCode, String cloudEvent) {
+                                                                                     int responseCode, String cloudEvent) {
```

Comment:
Why were spaces changed in that file ?

Reply:
Not sure exactly after rebase and space formatting , its leaving space

Reply:
Fixed now